spec/factories/lobby.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :lobby do
+    quiz
+    code { Faker::Lorem.word }
+    current_question_index { Faker::Number.between(from: 1, to: 5) } #quiz.questions.count + 1 - e bine si asa?
```

Comment:
I would definitely prefer the `quiz.questions.count + 1` instead of `5` since your lobby might not have 5 questions in total!